cmd/kubeadm/app/constants/constants.go

```diff
@@ -323,7 +323,7 @@ const (
 	CoreDNSDeploymentName = "coredns"
 
 	// CoreDNSImageName specifies the name of the image for CoreDNS add-on
-	CoreDNSImageName = "coredns"
+	CoreDNSImageName = "coredns/coredns"
```

Comment:
i just noticed this new sub path. 

could you please add the following second sentence in the release note:
> The v1.8.0 image is now located in a sub-path under k8s.gcr.io/coredns/coredns:v1.8.0

in case someone assumes the image is at `k8s.gcr.io/coredns:v1.18.0`

Reply:
When we make custom Kind node image, it does have an assumption on `k8s.gcr.io/coredns:1.8.0`,
now it breaks the basic workflow of making custom kind node image.

the question is why we are making a change on path? @rajansandeep cc @neolit123